pkg/translator/prometheus/README.md

```diff
@@ -74,6 +75,13 @@ List of standard OpenTelemetry units that will be translated to [Prometheus stan
 > **Note**
 > Prometheus also recommends using base units (no kilobytes, or milliseconds, for example) but these functions will not attempt to convert non-base units to base units.
 
+#### List of transformations performed to convert Prometheus metrics to OpenTelemetry metrics
+
+| Case                                                     | Transformation                                                     | Example                                                                         |
+|----------------------------------------------------------|--------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| Non-standard unit (unit is **not** surrounded with `{}`) | Drop the suffix, only if it's defined as UNIT in OpenMetric format | `system_network_dropped_packets` → `system_network_dropped` with unit `packets` |
```

Comment:
```suggestion
| Non-standard unit (unit is **not** surrounded with `{}`) | Drop the suffix if it's defined as a UNIT in the OpenMetrics format | `system_network_dropped_packets` → `system_network_dropped` with the unit `packets` |
```

Is there any list of all of the possible UNITs of the OpenMetrics format? I couldn't find one, but it'd be helpful to link here if there is.